Makefile

```diff
@@ -111,3 +111,23 @@ submodules:
 	@git submodule update --init --recursive
 
 .PHONY: manifests cobertura reviewable submodules fallthrough
+
+# ====================================================================================
+# Special Targets
+
+define CROSSPLANE_HELP
+Crossplane Targets:
+    manifests          Generate manifests e.g. CRD, RBAC etc.
+    cobertura          Generate a coverage report for cobertura applying exclusions on generated files.
+    reviewable         Ensure a PR is ready for review.
+    test-integration   Run integration tests.
+    submodules         Update the submodules, such as the common build scripts.
```

Comment:
Nitpick: the other sections have a blank line at the end, for visual separation between this section and the next section

Reply:
I'm not sold on the benefits, but thought this was nifty:

https://stackoverflow.com/questions/35730218/how-to-automatically-generate-a-makefile-help-command
(linked in the SO, https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html )

In either case, we miss the indirect `make` targets provided by the build/ submodule.